src/loaders/anndata-loaders/BaseAnnDataLoader.js

```diff
@@ -160,3 +160,12 @@ export default class BaseAnnDataLoader extends AbstractZarrLoader {
     return this.cellNames;
   }
 }
+
+
+export class DerivedAnnDataLoader {
+  /** @param {BaseAnnDataLoader} */
+  constructor(baseLoader) {
+    /** @type {BaseAnnDataLoader} */
```

Comment:
Not sure if this is "OK" for JSDoc annotations in this code base. I added this annotation so I could inspect `this.baseLoader` in the derived classes and make sure fields were present.